devices/philips.js

```diff
@@ -1065,7 +1073,6 @@ module.exports = [
         description: 'Hue white ambiance Aurelle square panel light',
         meta: {turnsOffAtBrightness1: true},
         extend: hueExtend.light_onoff_brightness_colortemp(),
-        ota: ota.zigbeeOTA,
```

Comment:
Why was this removed?

Reply:
The reason I removed is as follows.
1. To code, I talked with Sygnify(Philips Hue) but couldn’t get the proper information from them.
2. And they said this model, ie LTC005 is withdrawn from the market so no more development.
3. So I played with the definitions for other Synify models to come up with working set for LTC005
4. I selected 3216131P6 definition as the base and modified since the icon is similar to LTC005 
5. Since I couldn’t get no development/patch related information from Sygnify, I removed the OTA to prevent the wrong update.